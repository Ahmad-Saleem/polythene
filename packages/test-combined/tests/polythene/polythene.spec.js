import { tests } from "./tests.js";
import { runSnapshots } from "../../../test/scripts/snapshots";

runSnapshots(tests);
