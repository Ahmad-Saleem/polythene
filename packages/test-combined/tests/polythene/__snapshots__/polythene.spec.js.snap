exports[`test Button (theme: red) 1`] = `
"<a tabindex=\"0\" class=
\"pe-switch-control tests-polythene-themed-button\">
<div>
  <div class=\"pe-ripple pe-ripple--constrained\">
    <div class=\"pe-ripple__mask\">
      <div class=\"pe-ripple__waves\"></div>
    </div>
  </div>
  <div></div>
  <div></div>
  <div>
    Button
  </div>
</div></a>
"
`;

exports[`test Button 1`] = `
"<a tabindex=\"0\" class=\"pe-switch-control\">
<div>
  <div class=\"pe-ripple pe-ripple--constrained\">
    <div class=\"pe-ripple__mask\">
      <div class=\"pe-ripple__waves\"></div>
    </div>
  </div>
  <div></div>
  <div></div>
  <div>
    Button
  </div>
</div></a>
"
`;

exports[`test CSS Classes: blocks should be aligned vertically 1`] = `
"<div class=\"vertical-blocks\">
  <div class=\"block\">
    1
  </div>
  <div class=\"block\">
    2
  </div>
  <div class=\"block\">
    3
  </div>
  <div class=\"block\">
    4
  </div>
</div>
"
`;

exports[`test CSS Classes: blocks should be justified horizontally 1`] = `
"<div class=\"layout justified\">
  <div class=\"block\">
    1
  </div>
  <div class=\"block\">
    2
  </div>
  <div class=\"block\">
    3
  </div>
  <div class=\"block\">
    4
  </div>
</div>
"
`;

exports[`test Checkbox 1`] = `
"<div class=
\"pe-control pe-checkbox-control pe-control--on pe-control--regular\">
<input name=\"\" type=\"checkbox\" tabindex=\"-1\" class=
\"pe-control__input\"><label class=\"pe-control__form-label\"></label>
  <div class=\"pe-control__box\">
    <label class=\"pe-control__form-label\"></label>
    <div tabindex=\"0\" class=
    \"pe-button pe-icon-button pe-control__button\">
      <label class=\"pe-control__form-label\"></label>
      <div class=\"pe-button__content\">
        <label class=\"pe-control__form-label\"></label>
        <div class=\"pe-ripple pe-ripple--constrained\">
          <label class=\"pe-control__form-label\"></label>
          <div class=\"pe-ripple__mask\">
            <label class=\"pe-control__form-label\"></label>
            <div class=\"pe-ripple__waves\">
              <label class=\"pe-control__form-label\"></label>
            </div>
          </div>
        </div>
        <div class=\"pe-button__focus\">
          <label class=\"pe-control__form-label\"></label>
        </div>
        <div class=
        \"pe-icon pe-icon--regular pe-control__button--on\">
          <label class=\"pe-control__form-label\"></label>
          <div class=\"pe-svg\">
            <label class=\"pe-control__form-label\"></label>
            <svg xmlns=\"http://www.w3.org/2000/svg\" width=\"24\"
            height=\"24\" viewbox=\"0 0 24 24\">
              <label class=\"pe-control__form-label\"><path d=
              \"M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z\">
              </path></label>
            </svg>
          </div>
        </div>
        <div class=
        \"pe-icon pe-icon--regular pe-control__button--off\">
          <label class=\"pe-control__form-label\"></label>
          <div class=\"pe-svg\">
            <label class=\"pe-control__form-label\"></label>
            <svg xmlns=\"http://www.w3.org/2000/svg\" width=\"24\"
            height=\"24\" viewbox=\"0 0 24 24\">
              <label class=\"pe-control__form-label\"><path d=
              \"M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z\">
              </path></label>
            </svg>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div class=\"pe-control__label\">
    <label class=\"pe-control__form-label\">Label</label>
  </div>
</div>
"
`;

exports[`test FAB (theme: red) 1`] = `
"<a tabindex=\"0\" class=\"pe-fab tests-polythene-themed-fab\">
<div class=\"pe-button__content\">
  <div class=\"pe-shadow pe-shadow--animated\">
    <div class=\"pe-shadow__bottom pe-shadow--z-1\"></div>
    <div class=\"pe-shadow__top pe-shadow--z-1\"></div>
  </div>
  <div class=\"pe-ripple pe-ripple--constrained\">
    <div class=\"pe-ripple__mask\">
      <div class=\"pe-ripple__waves\"></div>
    </div>
  </div>
  <div class=\"pe-button__wash\"></div>
  <div class=\"pe-button__focus\"></div>
  <div class=\"pe-fab__content\">
    <div class=\"pe-icon pe-icon--regular\">
      <div class=\"pe-svg\">
        <svg xmlns=\"http://www.w3.org/2000/svg\" width=\"24\" height=
        \"24\" viewbox=\"0 0 24 24\">
          <path d=
          \"M3.9 12c0-1.71 1.39-3.1 3.1-3.1h4V7H7c-2.76 0-5 2.24-5 5s2.24 5 5 5h4v-1.9H7c-1.71 0-3.1-1.39-3.1-3.1zM8 13h8v-2H8v2zm9-6h-4v1.9h4c1.71 0 3.1 1.39 3.1 3.1s-1.39 3.1-3.1 3.1h-4V17h4c2.76 0 5-2.24 5-5s-2.24-5-5-5z\">
          </path>
        </svg>
      </div>
    </div>
  </div>
</div></a>
"
`;

exports[`test FAB 1`] = `
"<a tabindex=\"0\" class=\"pe-fab\">
<div class=\"pe-button__content\">
  <div class=\"pe-shadow pe-shadow--animated\">
    <div class=\"pe-shadow__bottom pe-shadow--z-1\"></div>
    <div class=\"pe-shadow__top pe-shadow--z-1\"></div>
  </div>
  <div class=\"pe-ripple pe-ripple--constrained\">
    <div class=\"pe-ripple__mask\">
      <div class=\"pe-ripple__waves\"></div>
    </div>
  </div>
  <div class=\"pe-button__wash\"></div>
  <div class=\"pe-button__focus\"></div>
  <div class=\"pe-fab__content\">
    <div class=\"pe-icon pe-icon--regular\">
      <div class=\"pe-svg\">
        <svg xmlns=\"http://www.w3.org/2000/svg\" width=\"24\" height=
        \"24\" viewbox=\"0 0 24 24\">
          <path d=
          \"M3.9 12c0-1.71 1.39-3.1 3.1-3.1h4V7H7c-2.76 0-5 2.24-5 5s2.24 5 5 5h4v-1.9H7c-1.71 0-3.1-1.39-3.1-3.1zM8 13h8v-2H8v2zm9-6h-4v1.9h4c1.71 0 3.1 1.39 3.1 3.1s-1.39 3.1-3.1 3.1h-4V17h4c2.76 0 5-2.24 5-5s-2.24-5-5-5z\">
          </path>
        </svg>
      </div>
    </div>
  </div>
</div></a>
"
`;

exports[`test Icon 1`] = `
"<div class=\"pe-icon pe-icon--regular\">
  <div class=\"pe-svg\">
    <svg xmlns=\"http://www.w3.org/2000/svg\" width=\"24\" height=\"24\"
    viewbox=\"0 0 24 24\">
      <path d=
      \"M3.9 12c0-1.71 1.39-3.1 3.1-3.1h4V7H7c-2.76 0-5 2.24-5 5s2.24 5 5 5h4v-1.9H7c-1.71 0-3.1-1.39-3.1-3.1zM8 13h8v-2H8v2zm9-6h-4v1.9h4c1.71 0 3.1 1.39 3.1 3.1s-1.39 3.1-3.1 3.1h-4V17h4c2.76 0 5-2.24 5-5s-2.24-5-5-5z\">
      </path>
    </svg>
  </div>
</div>
"
`;

exports[`test Icon Button 1`] = `
"<a tabindex=\"0\" class=\"pe-button pe-icon-button\">
<div style=\"background-color: rgb(255, 255, 255);\" class=
\"pe-button__content\">
  <div class=\"pe-ripple pe-ripple--constrained\">
    <div class=\"pe-ripple__mask\">
      <div class=\"pe-ripple__waves\"></div>
    </div>
  </div>
  <div class=\"pe-button__focus\"></div>
  <div class=\"pe-icon-button__content\">
    <div class=\"pe-icon pe-icon--regular\">
      <div class=\"pe-svg\">
        <svg xmlns=\"http://www.w3.org/2000/svg\" width=\"24\" height=
        \"24\" viewbox=\"0 0 24 24\">
          <path d=
          \"M3.9 12c0-1.71 1.39-3.1 3.1-3.1h4V7H7c-2.76 0-5 2.24-5 5s2.24 5 5 5h4v-1.9H7c-1.71 0-3.1-1.39-3.1-3.1zM8 13h8v-2H8v2zm9-6h-4v1.9h4c1.71 0 3.1 1.39 3.1 3.1s-1.39 3.1-3.1 3.1h-4V17h4c2.76 0 5-2.24 5-5s-2.24-5-5-5z\">
          </path>
        </svg>
      </div>
    </div>
  </div>
</div></a>
"
`;

exports[`test List 1`] = `
"<div class=\"pe-list pe-list--borders pe-list--header\">
  <div class=\"pe-list-tile pe-list__header\">
    <div style=\"\" class=\"pe-list-tile__primary\">
      <div class=\"pe-list-tile__content\">
        <div class=\"pe-list-tile__title\">
          Friends
        </div>
      </div>
    </div>
  </div>
  <div class=\"pe-list-tile pe-list-tile--subtitle\">
    <div style=\"\" class=\"pe-list-tile__primary\">
      <div class=
      \"pe-list-tile__content pe-list-tile__content--front\">
        <div class=\"pe-icon pe-icon--large pe-icon--avatar\">
          <img src=
          \"http://arthurclemens.github.io/assets/polythene/examples/avatar-1.png\">
        </div>
      </div>
      <div class=\"pe-list-tile__content\">
        <div class=\"pe-list-tile__title\">
          Jennifer Barker
        </div>
        <div class=\"pe-list-tile__subtitle\">
          Starting post doc
        </div>
      </div>
    </div>
  </div>
  <div class=\"pe-list-tile pe-list-tile--subtitle\">
    <div style=\"\" class=\"pe-list-tile__primary\">
      <div class=
      \"pe-list-tile__content pe-list-tile__content--front\">
        <div class=\"pe-icon pe-icon--large pe-icon--avatar\">
          <img src=
          \"http://arthurclemens.github.io/assets/polythene/examples/avatar-2.png\">
        </div>
      </div>
      <div class=\"pe-list-tile__content\">
        <div class=\"pe-list-tile__title\">
          Ali Connors
        </div>
        <div class=\"pe-list-tile__subtitle\">
          Brunch this weekend?
        </div>
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test List Tile 1`] = `
"<div class=\"pe-list-tile pe-list-tile--high-subtitle\">
  <div style=\"\" class=\"pe-list-tile__primary\">
    <div class=
    \"pe-list-tile__content pe-list-tile__content--front\">
      <div class=\"pe-icon pe-icon--large pe-icon--avatar\">
        <img src=
        \"http://arthurclemens.github.io/assets/polythene/examples/avatar-1.png\">
      </div>
    </div>
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
      <div class=
      \"pe-list-tile__subtitle pe-list-tile__high-subtitle\">
        The body lay naked and facedown, a deathly gray, spatters
        of blood staining the snow around it. It was minus fifteen
        degrees Celsius and a storm had passed just hours before.
      </div>
    </div>
  </div><a href=\"#!/\" class=\"pe-list-tile__secondary\">
  <div class=\"pe-list-tile__content\">
    <div class=\"pe-icon pe-icon--medium\">
      <div class=\"pe-svg\">
        <svg xmlns=\"http://www.w3.org/2000/svg\" width=\"24\" height=
        \"24\" viewbox=\"0 0 24 24\">
          <path d=
          \"M22 9.24l-7.19-.62L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.63-7.03L22 9.24zM12 15.4l-3.76 2.27 1-4.28-3.32-2.88 4.38-.38L12 6.1l1.71 4.04 4.38.38-3.32 2.88 1 4.28L12 15.4z\">
          </path>
        </svg>
      </div>
    </div>
  </div></a>
</div>
"
`;

exports[`test Menu 1`] = `
"<div class=
\"pe-menu pe-menu--permanent layout center-center pe-menu--width-5\">
  <div class=\"pe-menu__content\">
    <div class=\"pe-shadow pe-shadow--animated\">
      <div class=\"pe-shadow__bottom pe-shadow--z-1\"></div>
      <div class=\"pe-shadow__top pe-shadow--z-1\"></div>
    </div>
    <div class=\"pe-list pe-list--hoverable pe-list--compact\">
      <div class=\"pe-list-tile\">
        <div style=\"\" class=\"pe-list-tile__primary\">
          <div class=\"pe-list-tile__content\">
            <div class=\"pe-list-tile__title\">
              Bold
            </div>
          </div>
        </div>
        <div class=\"pe-list-tile__secondary\">
          <div class=\"pe-list-tile__content\">
            ⌘B
          </div>
        </div>
      </div>
      <div class=\"pe-list-tile\">
        <div style=\"\" class=\"pe-list-tile__primary\">
          <div class=\"pe-list-tile__content\">
            <div class=\"pe-list-tile__title\">
              Italic
            </div>
          </div>
        </div>
        <div class=\"pe-list-tile__secondary\">
          <div class=\"pe-list-tile__content\">
            ⌘I
          </div>
        </div>
      </div>
      <div class=\"pe-list-tile\">
        <div style=\"\" class=\"pe-list-tile__primary\">
          <div class=\"pe-list-tile__content\">
            <div class=\"pe-list-tile__title\">
              Underline
            </div>
          </div>
        </div>
        <div class=\"pe-list-tile__secondary\">
          <div class=\"pe-list-tile__content\">
            ⌘U
          </div>
        </div>
      </div>
      <div class=\"pe-list-tile\">
        <div style=\"\" class=\"pe-list-tile__primary\">
          <div class=\"pe-list-tile__content\">
            <div class=\"pe-list-tile__title\">
              Strikethrough
            </div>
          </div>
        </div>
        <div class=\"pe-list-tile__secondary\">
          <div class=\"pe-list-tile__content\">
            Alt+Shift+5
          </div>
        </div>
      </div>
      <div class=\"pe-list-tile\">
        <div style=\"\" class=\"pe-list-tile__primary\">
          <div class=\"pe-list-tile__content\">
            <div class=\"pe-list-tile__title\">
              Superscript
            </div>
          </div>
        </div>
        <div class=\"pe-list-tile__secondary\">
          <div class=\"pe-list-tile__content\">
            ⌘.
          </div>
        </div>
      </div>
      <div class=\"pe-list-tile\">
        <div style=\"\" class=\"pe-list-tile__primary\">
          <div class=\"pe-list-tile__content\">
            <div class=\"pe-list-tile__title\">
              Subscript
            </div>
          </div>
        </div>
        <div class=\"pe-list-tile__secondary\">
          <div class=\"pe-list-tile__content\">
            ⌘,
          </div>
        </div>
      </div>
    </div>
    <div class=\"pe-list pe-list--hoverable pe-list--compact\">
      <div class=\"pe-list-tile pe-list-tile--disabled\">
        <div style=\"\" class=\"pe-list-tile__primary\">
          <div class=\"pe-list-tile__content\">
            <div class=\"pe-list-tile__title\">
              Clear formatting
            </div>
          </div>
        </div>
        <div class=\"pe-list-tile__secondary\">
          <div class=\"pe-list-tile__content\">
            ⌘/
          </div>
        </div>
      </div>
      <div class=\"pe-list-tile\">
        <div style=\"\" class=\"pe-list-tile__primary\">
          <div class=\"pe-list-tile__content\">
            <div class=\"pe-list-tile__title\">
              Custom spacing
            </div>
          </div>
        </div>
        <div class=\"pe-list-tile__secondary\">
          <div class=\"pe-list-tile__content\"></div>
        </div>
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Radio button 1`] = `
"<div>
  <div class=
  \"pe-control pe-radio-control pe-control--off pe-control--regular\">
  <input name=\"radio\" type=\"radio\" tabindex=\"-1\" class=
  \"pe-control__input\"><label class=
  \"pe-control__form-label\"></label>
    <div class=\"pe-control__box\">
      <label class=\"pe-control__form-label\"></label>
      <div tabindex=\"0\" class=
      \"pe-button pe-icon-button pe-control__button\">
        <label class=\"pe-control__form-label\"></label>
        <div class=\"pe-button__content\">
          <label class=\"pe-control__form-label\"></label>
          <div class=\"pe-ripple pe-ripple--constrained\">
            <label class=\"pe-control__form-label\"></label>
            <div class=\"pe-ripple__mask\">
              <label class=\"pe-control__form-label\"></label>
              <div class=\"pe-ripple__waves\">
                <label class=\"pe-control__form-label\"></label>
              </div>
            </div>
          </div>
          <div class=\"pe-button__focus\">
            <label class=\"pe-control__form-label\"></label>
          </div>
          <div class=
          \"pe-icon pe-icon--regular pe-control__button--on\">
            <label class=\"pe-control__form-label\"></label>
            <div class=\"pe-svg\">
              <label class=\"pe-control__form-label\"></label>
              <svg xmlns=\"http://www.w3.org/2000/svg\" width=\"24\"
              height=\"24\" viewbox=\"0 0 24 24\">
                <label class=\"pe-control__form-label\"><path d=
                \"M12 7c-2.76 0-5 2.24-5 5s2.24 5 5 5 5-2.24 5-5-2.24-5-5-5zm0-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z\">
                </path></label>
              </svg>
            </div>
          </div>
          <div class=
          \"pe-icon pe-icon--regular pe-control__button--off\">
            <label class=\"pe-control__form-label\"></label>
            <div class=\"pe-svg\">
              <label class=\"pe-control__form-label\"></label>
              <svg xmlns=\"http://www.w3.org/2000/svg\" width=\"24\"
              height=\"24\" viewbox=\"0 0 24 24\">
                <label class=\"pe-control__form-label\"><path d=
                \"M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z\">
                </path></label>
              </svg>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class=\"pe-control__label\">
      <label class=\"pe-control__form-label\">Label</label>
    </div>
  </div>
  <div class=
  \"pe-control pe-radio-control pe-control--on pe-control--inactive pe-control--regular\">
  <input name=\"radio\" type=\"radio\" tabindex=\"-1\" class=
  \"pe-control__input\"><label class=
  \"pe-control__form-label\"></label>
    <div class=\"pe-control__box\">
      <label class=\"pe-control__form-label\"></label>
      <div tabindex=\"-1\" class=
      \"pe-button pe-icon-button pe-button--inactive pe-control__button\">
      <label class=\"pe-control__form-label\"></label>
        <div class=\"pe-button__content\">
          <label class=\"pe-control__form-label\"></label>
          <div class=\"pe-ripple pe-ripple--constrained\">
            <label class=\"pe-control__form-label\"></label>
            <div class=\"pe-ripple__mask\">
              <label class=\"pe-control__form-label\"></label>
              <div class=\"pe-ripple__waves\">
                <label class=\"pe-control__form-label\"></label>
              </div>
            </div>
          </div>
          <div class=\"pe-button__focus\">
            <label class=\"pe-control__form-label\"></label>
          </div>
          <div class=
          \"pe-icon pe-icon--regular pe-control__button--on\">
            <label class=\"pe-control__form-label\"></label>
            <div class=\"pe-svg\">
              <label class=\"pe-control__form-label\"></label>
              <svg xmlns=\"http://www.w3.org/2000/svg\" width=\"24\"
              height=\"24\" viewbox=\"0 0 24 24\">
                <label class=\"pe-control__form-label\"><path d=
                \"M12 7c-2.76 0-5 2.24-5 5s2.24 5 5 5 5-2.24 5-5-2.24-5-5-5zm0-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z\">
                </path></label>
              </svg>
            </div>
          </div>
          <div class=
          \"pe-icon pe-icon--regular pe-control__button--off\">
            <label class=\"pe-control__form-label\"></label>
            <div class=\"pe-svg\">
              <label class=\"pe-control__form-label\"></label>
              <svg xmlns=\"http://www.w3.org/2000/svg\" width=\"24\"
              height=\"24\" viewbox=\"0 0 24 24\">
                <label class=\"pe-control__form-label\"><path d=
                \"M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z\">
                </path></label>
              </svg>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class=\"pe-control__label\">
      <label class=\"pe-control__form-label\">Label</label>
    </div>
  </div>
</div>
"
`;

exports[`test Raised button (theme: red) 1`] = `
"<a tabindex=\"0\" class=
\"pe-button pe-text-button pe-raised-button tests-polythene-themed-raised-button\">
<div class=\"pe-button__content\">
  <div class=\"pe-shadow pe-shadow--animated\">
    <div class=\"pe-shadow__bottom pe-shadow--z-1\"></div>
    <div class=\"pe-shadow__top pe-shadow--z-1\"></div>
  </div>
  <div class=\"pe-ripple pe-ripple--constrained\">
    <div class=\"pe-ripple__mask\">
      <div class=\"pe-ripple__waves\"></div>
    </div>
  </div>
  <div class=\"pe-button__wash\"></div>
  <div class=\"pe-button__focus\"></div>
  <div class=\"pe-button__label\">
    Raised button
  </div>
</div></a>
"
`;

exports[`test Raised button 1`] = `
"<a tabindex=\"0\" class=\"pe-button pe-text-button pe-raised-button\">
<div class=\"pe-button__content\">
  <div class=\"pe-shadow pe-shadow--animated\">
    <div class=\"pe-shadow__bottom pe-shadow--z-1\"></div>
    <div class=\"pe-shadow__top pe-shadow--z-1\"></div>
  </div>
  <div class=\"pe-ripple pe-ripple--constrained\">
    <div class=\"pe-ripple__mask\">
      <div class=\"pe-ripple__waves\"></div>
    </div>
  </div>
  <div class=\"pe-button__wash\"></div>
  <div class=\"pe-button__focus\"></div>
  <div class=\"pe-button__label\">
    Raised button
  </div>
</div></a>
"
`;

exports[`test Ripple 1`] = `
"<div class=\"pe-ripple pe-ripple--constrained\">
  <div class=\"pe-ripple__mask\">
    <div class=\"pe-ripple__waves\"></div>
  </div>
</div>
"
`;

exports[`test SVG 1`] = `
"<div class=\"pe-svg\">
  <svg xmlns=\"http://www.w3.org/2000/svg\" width=\"24\" height=\"24\"
  viewbox=\"0 0 24 24\">
    <path d=
    \"M3.9 12c0-1.71 1.39-3.1 3.1-3.1h4V7H7c-2.76 0-5 2.24-5 5s2.24 5 5 5h4v-1.9H7c-1.71 0-3.1-1.39-3.1-3.1zM8 13h8v-2H8v2zm9-6h-4v1.9h4c1.71 0 3.1 1.39 3.1 3.1s-1.39 3.1-3.1 3.1h-4V17h4c2.76 0 5-2.24 5-5s-2.24-5-5-5z\">
    </path>
  </svg>
</div>
"
`;

exports[`test Shadow 1`] = `
"<div class=\"pe-shadow\">
  <div class=\"pe-shadow__bottom pe-shadow--z-1\"></div>
  <div class=\"pe-shadow__top pe-shadow--z-1\"></div>
</div>
"
`;

exports[`test Tabs 1`] = `
"<div class=\"pe-tabs pe-tabs--start pe-tabs--autofit\">
  <div class=\"pe-tabs__row\">
    <a tabindex=\"0\" class=
    \"pe-button pe-text-button pe-button--selected pe-tabs__tab\">
    <div class=\"pe-button__content\">
      <div class=\"pe-ripple pe-ripple--constrained\">
        <div class=\"pe-ripple__mask\">
          <div class=\"pe-ripple__waves\"></div>
        </div>
      </div>
      <div class=\"pe-button__focus\"></div>
      <div class=\"pe-tabs__tab-content\">
        <div class=\"pe-button__label\">
          <span>New</span>
        </div>
      </div>
    </div></a><a tabindex=\"0\" class=
    \"pe-button pe-text-button pe-tabs__tab\">
    <div class=\"pe-button__content\">
      <div class=\"pe-ripple pe-ripple--constrained\">
        <div class=\"pe-ripple__mask\">
          <div class=\"pe-ripple__waves\"></div>
        </div>
      </div>
      <div class=\"pe-button__focus\"></div>
      <div class=\"pe-tabs__tab-content\">
        <div class=\"pe-button__label\">
          <span>My Favorites</span>
        </div>
      </div>
    </div></a><a tabindex=\"0\" class=
    \"pe-button pe-text-button pe-tabs__tab\">
    <div class=\"pe-button__content\">
      <div class=\"pe-ripple pe-ripple--constrained\">
        <div class=\"pe-ripple__mask\">
          <div class=\"pe-ripple__waves\"></div>
        </div>
      </div>
      <div class=\"pe-button__focus\"></div>
      <div class=\"pe-tabs__tab-content\">
        <div class=\"pe-button__label\">
          <span>Saved</span>
        </div>
      </div>
    </div></a>
    <div class=\"pe-tabs__indicator\"></div>
  </div>
</div>
"
`;

exports[`test Toolbar 1`] = `
"<div class=\"pe-toolbar\">
  <a tabindex=\"0\" class=\"pe-button pe-icon-button\">
  <div class=\"pe-button__content\">
    <div class=\"pe-ripple pe-ripple--constrained\">
      <div class=\"pe-ripple__mask\">
        <div class=\"pe-ripple__waves\"></div>
      </div>
    </div>
    <div class=\"pe-button__focus\"></div>
    <div class=\"pe-icon-button__content\">
      <div class=\"pe-icon pe-icon--regular\">
        <div class=\"pe-svg\">
          <svg xmlns=\"http://www.w3.org/2000/svg\" width=\"24\"
          height=\"24\" viewbox=\"0 0 24 24\">
            <path d=
            \"M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z\"></path>
          </svg>
        </div>
      </div>
    </div>
  </div></a><span>Title</span><a tabindex=\"0\" class=
  \"pe-button pe-icon-button\">
  <div class=\"pe-button__content\">
    <div class=\"pe-ripple pe-ripple--constrained\">
      <div class=\"pe-ripple__mask\">
        <div class=\"pe-ripple__waves\"></div>
      </div>
    </div>
    <div class=\"pe-button__focus\"></div>
    <div class=\"pe-icon-button__content\">
      <div class=\"pe-icon pe-icon--regular\">
        <div class=\"pe-svg\">
          <svg xmlns=\"http://www.w3.org/2000/svg\" width=\"24\"
          height=\"24\" viewbox=\"0 0 24 24\">
            <path d=\"M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z\"></path>
          </svg>
        </div>
      </div>
    </div>
  </div></a>
</div>
"
`;

exports[`test Web font loader (Adobe Edge Web Fonts: Patua One) 1`] = `
"<p class=\"patua-one\">
  The sky was cloudless and of a deep dark blue.
</p>
"
`;

exports[`test Web font loader (Google Fonts: PT Sans) 1`] = `
"<p class=\"pt-sans\">
  The sky was cloudless and of a deep dark blue.
</p>
"
`;
